sdk/core/azure_core_amqp/src/error.rs

```diff
@@ -73,6 +145,31 @@ impl AmqpError {
     pub fn kind(&self) -> &AmqpErrorKind {
         &self.kind
     }
+
+    #[cfg(feature = "test")]
+    pub fn new_management_error(
+        status_code: azure_core::http::StatusCode,
+        description: Option<String>,
+    ) -> Self {
+        Self {
+            kind: AmqpErrorKind::ManagementStatusCode(status_code, description),
+        }
+    }
+
+    #[cfg(feature = "test")]
```

Comment:
Tip (and I'm not advocating you change this, but for next time): if you ever find yourself wanting to condition a bunch of functions, have a separate `impl MyType` (well, if methods) with the `#[cfg]` on it and then all the functions therein are conditioned for free.

Reply:
Oh! I didn't realize you could have multiple `impl MyType` blocks.